_includes/config-guide/custom-logger-handler-examples.md

```diff
@@ -0,0 +1,157 @@
+You can use one of the following approaches for logging into custom file:
+- setup custom log file in the `di.xml`
+- setup custom file in the custom logger handler class 
+
+## Setup custom log file in the `di.xml`
+
+1. Define custom log file in the virtual type
+
+```xml
+<virtualType name="Magento\Payment\Model\Method\VirtualDebug" type="Magento\Framework\Logger\Handler\Base">
+    <arguments>
+        <argument name="fileName" xsi:type="string">/var/log/payment.log</argument>
+    </arguments>
+</virtualType>
+```
+
+1. Specify handler which will be used
+
+```xml
+<virtualType name="Magento\Payment\Model\Method\VirtualLogger" type="Magento\Framework\Logger\Monolog">
+    <arguments>
+        <argument name="handlers" xsi:type="array">
+            <item name="debug" xsi:type="object">Magento\Payment\Model\Method\VirtualDebug</item>
+        </argument>
+    </arguments>
+</virtualType>
+```
+
+1. Inject class of virtual logger to `$this->logger`
+
+```xml
+<type name="Magento\Payment\Model\Method\Logger">
+    <arguments>
+        <argument name="logger" xsi:type="object">Magento\Payment\Model\Method\VirtualLogger</argument>
+    </arguments>
+</type>
+```
+
+1. Virtual class `Magento\Payment\Model\Method\VirtualLogger` will be injected into `$this->logger` and will log exception messages into `/var/log/payment.log` file. 
```

Comment:
```suggestion
1. Virtual class `Magento\Payment\Model\Method\VirtualLogger` will be injected into the class (`Magento\Payment\Model\Method\Logger` in the example above) and will log exception messages into `/var/log/payment.log` file. 
```
Basically, we cannot be 100% sure that it will be `$this->logger`. We will inject it as a constructor argument with the name `$logger` but the class property name can be declared in the constructor like `$this->errorLogger = $logger`

Reply:
Yes, you are correct here, @rogyar !

Custom handler class will be injected into error handler which we specify in `di.xml`.

I've fixed information about that.